lib/std/os/linux.zig

```diff
@@ -1227,3 +1227,464 @@ test "" {
         _ = @import("linux/test.zig");
     }
 }
+
+pub const bpf = struct {
+    /// a single BPF instruction
+    pub const Insn = packed struct {
```

Comment:
Struct and enum definitions have so far gone into os/bits/linux.zig rather than os/linux.zig

Reply:
I forgot that file existed, I'll also work to make it more consistent with the name/type conventions in the file